backend/apps/search/tasks.py

```
@@ -53,6 +57,23 @@ def index_model_for_search(
+        if index:
+            index.add_documents([
+                {
+                    "id": str(doc.id),
+                    "title": doc.title,
+                    "description": doc.description,
+                    "tags": doc.tags,
+                    "body_text": doc.body_text,
+                    "content_type_name": doc.content_type_name,
+                }
+            ])
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)